packages/filebrowser/src/listing.ts

```diff
@@ -860,6 +882,22 @@ class DirListing extends Widget {
     default:
       break;
     }
+
+    // Detects printable characters typed by the user.
+    // Not all browsers support .key, but it discharges us from reconstructing
+    // characters from key codes.
+    if (event.key !== undefined && event.key.length === 1) {
```

Comment:
Ah, you are correct.  We had been using this for keyboard shortcuts, where the modifier is specified separately.